client-js/src/client/errors.js

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright 2018, TeamDev. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+import 'spine-web-client-proto/spine/base/error_pb';
+// noinspection JSUnresolvedVariable
+const SpineBaseError = proto.spine.base.Error;
```

Comment:
Why this instead and not import as?

Reply:
```
import {Error as ImportedClass} from 'spine-web-client-proto/spine/base/error_pb';
const ClassFromNamespace = proto.spine.base.Error;

error instanceof ImportedClass-> false
error instanceof ClassFromNamespace -> true
```
When I import `spine.base.Error` constructor with module import statement and use alias (`as ImportedError`),  the `instanceof` operator returns `false` with instance of `spine.base.Error`. Module import wraps it or makes a copy, I'm not sure. You can work with this constructor as you expect (make instances etc.), but `instanceof` fails with it.

Importing of the constructor directly without an alias `import {Error} from` would help, but the `Error` name is occupied with the canonical Js Error which is used in this module.